vimeo/vimeo-networking-java
# ANDTV-78 Player Abstraction Part 1

#### Ticket
[ANDTV-78](https://vimean.atlassian.net/browse/ANDTV-78)
[ANDTV-102](https://vimean.atlassian.net/browse/ANDTV-102)
[ANDTV-82](https://vimean.atlassian.net/browse/ANDTV-82) 
[ANDTV-84](https://vimean.atlassian.net/browse/ANDTV-84)

#### Ticket Summary
The original ticket is about playback - these changes to networking are mostly just optimizations.

#### Implementation Summary
* Made Gson a single instance
* Added 1.7 compatibility so I could compile with Java 8 and not have networking wig out
   * This will change when we go to Java 8
* Added in some convenience methods for VOD related getters
* Removed the need for our ISO8601.java class for deserialization
   * It looks like Gson added support for this a while back (after we added it in)

#### How to Test
* Make sure deserialization works
* Make sure you can build with Java 7 still
